src-built-in/components/appLauncher2/src/components/FilterSort.jsx

```diff
@@ -1,25 +1,42 @@
 import './FilterSort.css'
 import React from  'react'
+import SearchBox from './SearchBox'
+import TagsMenu from './TagsMenu'
+import SortBy from './SortBy'
+import {getStore} from '../stores/LauncherStore'
 
 export default class FilterSort extends React.Component {
 
 	constructor(props) {
 		super(props)
+		this.state = {
+			search: ''
+		}
+		this.onSearch = this.onSearch.bind(this)
+	}
+	onSearch(event) {
+		this.setState({
+			search: event.target.value
+		}, () => {
+			getStore().setValue({
+				field: 'filterText', 
+				value: this.state.search
+			})
+		})
+	}
+
+	onItemClick(item) {
```

Comment:
Make sure you install some kind of linter. the single quotes will get replaced with double quotes, formatting will fix itself. I highly recommend VSCode.